include/integratorxx/quadratures/detail.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
```

Comment:
The name of this header is not descriptive. Something like `quadratures/detail/copy_grid.hpp` would suffice. This also allows for IWYU later on.

Reply:
Changed name